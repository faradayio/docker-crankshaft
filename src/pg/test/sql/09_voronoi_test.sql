\pset format unaligned
\set ECHO none

WITH a AS (
    SELECT
    ARRAY[ST_GeomFromText('POINT(2.1744 41.403)', 4326),ST_GeomFromText('POINT(2.1228 41.380)', 4326),ST_GeomFromText('POINT(2.1511 41.374)', 4326),ST_GeomFromText('POINT(2.1528 41.413)', 4326),ST_GeomFromText('POINT(2.165 41.391)', 4326),ST_GeomFromText('POINT(2.1498 41.371)', 4326),ST_GeomFromText('POINT(2.1533 41.368)', 4326),ST_GeomFromText('POINT(2.131386 41.41399)', 4326)] AS geomin
),
b as(
    SELECT
        (st_dump(cdb_crankshaft.CDB_voronoi(geomin, 0.2, 1e-9))).geom as result
    FROM a
)
SELECT
    abs(avg(st_area(result)) - 0.000178661700690617) < 1e-6 as within_tolerance
FROM b;
