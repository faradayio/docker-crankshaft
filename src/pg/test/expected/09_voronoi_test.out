\pset format unaligned
\set ECHO none
within_tolerance
t
(1 row)
